cl/corpus_importer/import_columbia/columbia_utils.py

```diff
@@ -0,0 +1,841 @@
+import itertools
+import re
+from datetime import date
+from typing import Any, Optional
+
+import dateutil.parser as dparser
+from bs4 import BeautifulSoup, NavigableString, Tag
+from juriscraper.lib.string_utils import clean_string, harmonize, titlecase
+
+from cl.people_db.lookup_utils import extract_judge_last_name
+
+FILED_TAGS = [
+    "filed",
+    "opinion filed",
+    "date",
+    "order filed",
+    "delivered and filed",
+    "letter filed",
+    "dated",
+    "release date",
+    "filing date",
+    "filed date",
+    "date submitted",
+    "as of",
+    "opinions filed",
+    "filed on",
+    "decision filed",
+    "date delivered",
+    "affirmed and opinion filed",
+    "dismissed and opinion filed",
+    "decided and entered",
+]
+DECIDED_TAGS = ["decided", "date decided", "decided on", "decided date"]
+ARGUED_TAGS = [
+    "argued",
+    "submitted",
```

Comment:
This code is duplicative of no?  Its all in the populate opinions file

Reply:
No, the first thing i did was update the columbia importer, in that PR i deleted almost all those files in the importer PR #2865 , i moved all the code required to corpus_importer/utils.py and  corpus_importer/import_columbia/columbia_utils.py

Reply:
Are you sure?